properties.schema

```diff
@@ -511,140 +521,140 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#263944",
-            "help": "Drawer background colour"
+            "help": "Sets the Drawer (slide out menu) colour."
```

Comment:
I'm not sure this help text really adds anything as the title is sufficiently clear by itself.